Vagrantfile

```diff
@@ -6,8 +6,14 @@ VAGRANTFILE_API_VERSION = '2'
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = 'debian/buster64'
 
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 4096
+    v.cpus = 2
+  end
+
   config.vm.synced_folder '.', '/vagrant', type: 'rsync',
-    rsync__exclude: ['node_modules/', 'client/dist/']
+    rsync__exclude: ['node_modules/', 'client/dist/'],
+    rsync__args: ['-r', '--perms', '--chmod=Du=rwx,Dg=rwx,Do=rwx,Fu=rwx,Fg=rwx,Fo=rwx', './server']
```

Comment:
@jugglinmike This is a naive world writable solution for the rsync options on the server folder. I know it's not setting the ownership on `aria-bot`, as suggested, but it does circumvent a Vagrant error on rsync when `aria-bot` isn't created on first provision. Given the time constraints, could we push this or something like this forward so that we can get the upgrade in place before the sprint ends?

Reply:
@jugglinmike I can also confirm that with this change, I am no longer running into permissions issues.